lib/src/subscription.rs

```diff
@@ -0,0 +1,22 @@
+use yrs::Subscription;
+
+pub(crate) struct YSubscription {
+  #[allow(dead_code)]
+  value: Subscription
+}
+
+impl YSubscription {
+  pub(crate)fn new(value: Subscription) -> YSubscription {
+      // println!("YSubscription is being initialized");
+
+      YSubscription {
+        value
+      }
+  }
+}
+
+// impl Drop for YSubscription {
```

Comment:
I can delete this. This is when I was testing that this was being properly deinitialized.

Reply:
yeah, go ahead and drop it if you're not going to be using it down the road.